tests/fixtures/mod.rs

```diff
@@ -45,6 +45,13 @@ pub static DEEPLY_NESTED_FILE: &str = "very/deeply/nested/test.rs";
 /// Name of a symlink pointing to a directory
 pub static DIRECTORY_SYMLINK: &str = "dir_symlink/";
 
+/// Name of a directory inside a symlinked directory
+#[allow(unused)]
+pub static SYMLINK_NESTED_DIR: &str = "dir_symlink/nested";
+
+/// Name of a file inside a directory inside a symlinked directory
+pub static SYMLINK_NESTED_FILE: &str = "dir_symlink/nested/file";
```

Comment:
Maybe make the names even more obvious? `DIR_BEHIND_SYMLINKED_DIR` and `FILE_NESTED_BEHIND_SYMLINKED_DIR` perhaps.